contracts/MockToken.sol

```diff
@@ -0,0 +1,126 @@
+pragma solidity ^0.4.23;
+
+// Copyright 2017 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// 
+// ----------------------------------------------------------------------------
+// Contracts: MockToken 
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+
+import "./EIP20Interface.sol";
+import "./MockTokenConfig.sol";
+import "./OpsManaged.sol";
+import "./SafeMath.sol";
+
+contract MockToken is EIP20Interface, MockTokenConfig, OpsManaged {
```

Comment:
`MockToken` does not need `OpsManaged`. `OpsManaged` should be replaced with `Owner`.